app/src/main/resources/conf-camel-rabbitmq.yml

```diff
@@ -3,3 +3,6 @@ message-queue:
   port: 5672
   user: "${RABBITMQ_PLACEHOLDERS_USERNAME:guest}"
   password: "${RABBITMQ_PLACEHOLDERS_USERPASSWORD:guest}"
+
+mas-processing-initial-delay: 2000
+mas-processing-subsequent-delay: 1000
```

Comment:
Can you make this environment variables with these defaults? I would like to test with longer times to make sure timeouts are working?

Reply:
ok done